cgnsutilities/cgnsutilities.py

```diff
@@ -988,7 +987,7 @@ def func(P):
                     x0bin = xmin + dxBin * binIndex
                     xfbin = xmin + dxBin * (binIndex + 1)
                     # Find cells that touch this interval and get their edges
-                    bol = -(((S[:-1] < x0bin) * (S[1:] < x0bin)) + ((S[:-1] > xfbin) * (S[1:] > xfbin)))
+                    bol = ~(((S[:-1] < x0bin) * (S[1:] < x0bin)) + ((S[:-1] > xfbin) * (S[1:] > xfbin)))
```

Comment:
I don't understand this change.

Reply:
This is essentially inverting a boolean array. This is a bit cryptic, and perhaps numpys `invert` would be more readable (see [docs here](https://numpy.org/doc/stable/reference/generated/numpy.invert.html)).

Reply:
Oh ok yeah, it is a bit cryptic, you mean flipping the boolean values element-by-element? Could you just expand the comment in the code? Otherwise the PR is ready to go for me.

Reply:
I added a comment on this, but not sure if this is useful. Please take a look. In any case, Ney suggests that this is not a useful function, and we should perhaps deprecate in favor of `explicitCart` and `simpleCart`.